packages/cli/src/databases/migrations/common/1711390882123-MoveSshKeysToDatabase.ts

```diff
@@ -57,7 +57,7 @@ export class MoveSshKeysToDatabase1711390882123 implements ReversibleMigration {
 		});
 
 		await runQuery(
-			`INSERT INTO ${settings} (key, value) VALUES ('${this.settingsKey}', '${value}');`,
+			`INSERT INTO ${settings} (\`key\`, \`value\`) VALUES ('${this.settingsKey}', '${value}');`,
```

Comment:
I'm afraid this might now work properly for MySQL but not for Postgres or SQLite - have you tried running the database test pipeline on this PR? Just to make sure.

Reply:
CI passed: https://github.com/n8n-io/n8n/actions/runs/8643099792

Reply:
CI is passing likely because these queries are not running there, because there are no ssh keys to migrate. 
We should use `escape.columnName` to wrap column names instead.